QUICKSTART.md

```diff
@@ -0,0 +1,354 @@
+# **KubeStellar** Quickstart
+
+## Required Packages:
+   - [docker](https://docs.docker.com/engine/install/)
```

Comment:
We should support use of free alternatives to Docker.
Mac users in medium and large businesses can not use Docker for free.
https://docs.docker.com/engine/install tells the mac user to use "Docker Desktop for Mac" and refers to https://docs.docker.com/desktop/install/mac-install/ , which includes this remark:

> Commercial use of Docker Desktop in larger enterprises (more than 250 employees OR more than $10 million USD in annual revenue) requires a paid subscription.

Reply:
Yes, I agree that we should further explore about using another alternatives of Docker, but I believe that at this moment we are not ready to add any information in the quickstart regarding to that, maybe in the next release??

Reply:
I have been testing with Podman and think that we can say that works.

Reply:
Great, could you provide a link for podman to add in the quickstart?

Reply:
We might update this entry to something like the following.

- [docker](https://docs.docker.com/engine/install/) or something close enough that appears on your `$PATH` as `docker`.  [podman](https://podman.io/) qualifies.

Reply:
for this release we are not mentioning Podman.  Let's put that in a PR for the next version of the quickstart.  Just making an executive decision here so that we can converge and complete this PR.

Reply:
resolving